src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -216,11 +216,12 @@
       <PostDeploymentScriptArgument>@(PostDeploy->'--postdeploy %(Identity)', ' ')</PostDeploymentScriptArgument>
       <RefactorLogScriptArgument>@(RefactorLog->'--refactorlog %(Identity)', ' ')</RefactorLogScriptArgument>
       <DebugArgument Condition="'$(MSBuildSdkSqlProjDebug)' == 'True'">--debug</DebugArgument>
-      <TreatTSqlWarningsAsErrorsArgument Condition="'$(TreatTSqlWarningsAsErrors)' == 'True' Or '$(TreatWarningsAsErrors)' == 'True'">--warnaserror</TreatTSqlWarningsAsErrorsArgument>
+      <TreatTSqlWarningsAsErrorsArgument Condition="'$(TreatTSqlWarningsAsErrors)' == 'True' Or ('$(TreatWarningsAsErrors)' == 'True' And '$(TreatTSqlWarningsAsErrors)' == '')">--warnaserror</TreatTSqlWarningsAsErrorsArgument>
```

Comment:
In the interests of a clean commit history that's aligned with individual issues, I think this change should be submitted in a separate PR. That's my personal preference, not sure if @jmezach or @ErikEJ have strong feelings either way. 🙂

Reply:
Makes sense. I'll separate the commits ;-)

Reply:
Please revert this and do a separate PR

Reply:
I thought I did 😶